R/pkg/R/DataFrame.R

```diff
@@ -2973,15 +2974,51 @@ setMethod("describe",
             dataFrame(sdf)
           })
 
+#' summary
+#'
+#' Computes specified statistics for numeric and string columns.
+#'
+#' Available statistics are:
+#'
+#' - count
+#' - mean
+#' - stddev
+#' - min
+#' - max
+#' - arbitrary approximate percentiles specified as a percentage (eg, 75%)
+#'
+#' If no statistics are given, this function computes count, mean, stddev, min,
+#' approximate quartiles (percentiles at 25%, 50%, and 75%), and max.
```

Comment:
also, don't use empty line - like #' - the 2nd one becomes the "details" section in the doc as formatted by roxygen2